### ISSUE TITLE
Default language doesn't reside on it's own folder

### ISSUE THREAD

**Issue by andreamoro** [Anthony-Gaudino/jekyll-multiple-languages-plugin#103]
Apologies for repeating something known if any, but I'm fairly new to Jekyll.

I need the default language - which I assume to be the first one listed in the collection - to reside under it's own separate folder.
In my case I've a ["it", "en"] collection, but only the "en" subfolder is created under the _site subfolder.

Any idea?

**Comment by emmerich** [comment]
Yep, I had the same problem. It's an implementation detail of the library. I got around by specifying:

`["en", "en", "fr"]`

This generates a folder `en` and `fr`, and also generates the "default" files as `en`. Hope this helps.

**Comment by ygd** [comment]
would be cool if there is a more elegant way to handle this. 

eg. you have `subfolder_for_default_lang: true` in your config and it generates subfolders for all languages. Nice would be if `/` would redirect to `/default_lang` via `<meta http-equiv="refresh" content="0; URL=/default_lang/">`

**Comment by Levino** [comment]
All languages should be in subfolders. It makes no sense to have a "special role" for the default language. One should use forwarding based on user agent from the domain.com/home to domain.com/language/home.

**Comment by bpatra** [comment]
Could be duplicated with #86
